### PR TITLE
Move code from service/common/telemetry to the existing telemetry pkg

### PR DESCRIPTION
**Pull request by michaljurecko**
Part of: https://keboola.atlassian.net/browse/BA-38

### CODE REVIEW

File: internal/pkg/service/buffer/api/dependencies/dependencies.go [hunk 1]
```diff
@@ -40,8 +40,7 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/env"
 	"github.com/keboola/keboola-as-code/internal/pkg/log"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/common/dependencies"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/common/telemetry"
-	telemetryUtils "github.com/keboola/keboola-as-code/internal/pkg/telemetry"
+	"github.com/keboola/keboola-as-code/internal/pkg/telemetry"
```
Comment (michaljurecko) [inline internal/pkg/service/buffer/api/dependencies/dependencies.go:43]: Mali sme zbytocne 2 `telemetry` pkgs, v kode to robilo chaos.

File: internal/pkg/telemetry/ddhttp.go [hunk 2]
```diff
@@ -0,0 +1,117 @@
+package telemetry
```
Comment (michaljurecko) [inline internal/pkg/telemetry/ddhttp.go:1]: kod presunuty bez zmeny, potriedeny do samostatnych suborov.

File: internal/pkg/telemetry/ddhttp.go [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+	"time"
+
+	"github.com/keboola/go-client/pkg/client"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	ddtracer "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/strhelper"
+)
+
+func HttpClientTraceFactory() client.TraceFactory {
```
Comment (JakubMatejka) [inline internal/pkg/telemetry/ddhttp.go:17]: `HTTPClientTraceFactory`? 😄

Comment (michaljurecko) [reply internal/pkg/telemetry/ddhttp.go:17]: Ako to mam nazvat? :slightly_smiling_face:  `DDTraceFactory` znie lepsie

Comment (michaljurecko) [reply internal/pkg/telemetry/ddhttp.go]: https://github.com/keboola/keboola-as-code/pull/999/commits/902332719643821b05aee515c4a662d084d0b72a

**Review by JakubMatejka — APPROVED** [review]
🙂

`#999`❗

**Pull request merged**